packages/utils/src/cli-utils/findBinPath.test.ts

```diff
@@ -40,15 +40,6 @@ describe('findBinPath', () => {
     expect(binPath).toEqual(cmdPath);
   });
 
-  it('should find bin path two dir up', () => {
-    const base = join(__dirname, '..', '..'); // two dirs up
-    const { cmdName, cmdPath, resetDisk } = bootstrap(base);
-    const binPath = findBinPath(cmdName, base);
-
-    resetDisk();
-    expect(binPath).toEqual(cmdPath);
-  });
-
```

Comment:
Did you remove this test by mistake?

Reply:
@arboleya this unit test is the source of the problem. 

It is deleting the node_modules from the utils package. 

That's why running the tests 2 times in a row result in an error.